arts/observation.py

```diff
@@ -56,3 +60,25 @@ def beamsize(self):
 
         """
         return 0
+
+    def dm_to_dist(self, ra, dec, dm, model):
+        """
+
+        Args:
+            ra (string): right ascension of source
+            dec (string): declination of source
+            dm (float): dispersion measure (pc/cm^3)
+            model (string): electron density model; options are ne2001 and ymw16
+
+        Returns:
+            dist (float): calculated distance to the source
+
+        """
+        coords = SkyCoord(ra, dec, frame="icrs")
+        coords = coords.galactic
+        l, b = coords.l, coords.b
+        # convert to galactic coordinates
+
+        dist, tau_sc = pygedm.dm_to_dist(l, b, dm, method=model)
+
+        return dist
```

Comment:
Does this return a float or an astropy object?

Reply:
Ah, yes, you're right - I've changed it to return a float!